src/shell/dotnet/Shell/Utilities/ResourceReader.cs

```diff
@@ -13,28 +13,21 @@
  */
 
 using System;
-using System.Collections.Generic;
 using System.IO;
-using System.Linq;
 using System.Reflection;
-using System.Text;
-using System.Threading.Tasks;
-using System.Windows;
 
-namespace Shell.Utilities
+namespace Shell.Utilities;
+
+public static class ResourceReader
 {
-    public static class ResourceReader
+    public static string ReadResource(string resourcePath)
     {
-        public static string ReadResource(string name)
-        {
-            var assembly = Assembly.GetExecutingAssembly();
-            string resourcePath = name;
+        var assembly = Assembly.GetExecutingAssembly();
 
-            using var stream = assembly.GetManifestResourceStream(resourcePath) ?? throw new InvalidOperationException("Resource not found");
-            using var reader = new StreamReader(stream);
-            {
-                return reader.ReadToEnd();
-            }
+        using var stream = assembly.GetManifestResourceStream(resourcePath) ?? throw new InvalidOperationException("Resource not found");
+        using var reader = new StreamReader(stream);
+        {
```

Comment:
Nitpick: { } are unnecessary here

Reply:
It is a merged code, I didn't want to change that much :) but then I'll remove the {}